### PR TITLE
demo: remove emotion

### PR DESCRIPTION
**Pull request by linxianxi**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [x] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Workflow
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |      -     |
| 🇨🇳 Chinese |     -      |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

---

### <samp>🤖 Generated by Copilot at 647e50d</samp>

Improved the performance and readability of the draggable tabs and upload demos by using `react-sortable-hoc` and removing redundant code. Moved some CSS rules from the component level to the global style sheet to avoid unwanted interactions.

### <samp>🤖 Generated by Copilot at 647e50d</samp>

*  Simplify the logic and animation of the `DraggableTabNode` component by removing the `useEffect` hook, the `className` state, and the `onActiveBarTransform` prop ([link](https://github.com/ant-design/ant-design/pull/43682/files?diff=unified&w=0#diff-712c298e62aec8eeacb8d33bd8bab10a0d246ef5960b16e444e26f6ba0be7d80L10-R18),[link](https://github.com/ant-design/ant-design/pull/43682/files?diff=unified&w=0#diff-712c298e62aec8eeacb8d33bd8bab10a0d246ef5960b16e444e26f6ba0be7d80L31-L45),[link](https://github.com/ant-design/ant-design/pull/43682/files?diff=unified&w=0#diff-712c298e62aec8eeacb8d33bd8bab10a0d246ef5960b16e444e26f6ba0be7d80L73-L74),[link](https://github.com/ant-design/ant-design/pull/43682/files?diff=unified&w=0#diff-712c298e62aec8eeacb8d33bd8bab10a0d246ef5960b16e444e26f6ba0be7d80L89),[link](https://github.com/ant-design/ant-design/pull/43682/files?diff=unified&w=0#diff-712c298e62aec8eeacb8d33bd8bab10a0d246ef5960b16e444e26f6ba0be7d80L96-R76))
* Move the `SortableContext` component to wrap the `Tabs` component and set the `strategy` prop to `horizontalListSortingStrategy` to handle the sorting logic and animation of the tabs ([link](https://github.com/ant-design/ant-design/pull/43682/files?diff=unified&w=0#diff-712c298e62aec8eeacb8d33bd8bab10a0d246ef5960b16e444e26f6ba0be7d80L31-L45))
* Move the CSS code that sets the `pointer-events` to `none` for the anchor elements inside the `.is-dragging` class from the `DraggableUploadListItem` component to the global style sheet in `components/upload/demo/drag-sorting.md` to prevent the preview event from triggering when the user drags an upload item ([link](https://github.com/ant-design/ant-design/pull/43682/files?diff=unified&w=0#diff-5381d06d4f27d1e8c949591762333b51dc80b68802bdd025f4451d81cd827ccbR8-R13),[link](https://github.com/ant-design/ant-design/pull/43682/files?diff=unified&w=0#diff-1141c1ad5641407286c16c966c24a7ae1da8a43b84e1829f6c68192e8aaeb557L11),[link](https://github.com/ant-design/ant-design/pull/43682/files?diff=unified&w=0#diff-1141c1ad5641407286c16c966c24a7ae1da8a43b84e1829f6c68192e8aaeb557L32-R39))
* Add the `is-dragging` class to the `className` prop of the `DraggableUploadListItem` component conditionally based on the `isDragging` value ([link](https://github.com/ant-design/ant-design/pull/43682/files?diff=unified&w=0#diff-1141c1ad5641407286c16c966c24a7ae1da8a43b84e1829f6c68192e8aaeb557L32-R39))
* Move the `SortableContext` component to wrap the `Upload.Dragger` component and change the `strategy` prop from `rectSortingStrategy` to `verticalListSortingStrategy` to suit the vertical list layout of the upload items ([link](https://github.com/ant-design/ant-design/pull/43682/files?diff=unified&w=0#diff-1141c1ad5641407286c16c966c24a7ae1da8a43b84e1829f6c68192e8aaeb557L104-R101))

### CODE REVIEW

**Comment by yoyo837** [comment]
这个PR的 body, 好像changelog不能省略

**Comment by linxianxi** [comment]
> 这个PR的 body, 好像changelog不能省略

是么，我之前改 demo changelog 都是填 `-`

**Comment by yoyo837** [comment]
> > 这个PR的 body, 好像changelog不能省略
> 
> 是么，我之前改 demo changelog 都是填 `-`

一遍不需要写入发布的changelog的就懒得写了, 这个也不用写入发布的话, 那就不写

File: components/upload/demo/drag-sorting.tsx [hunk 1]
```diff
@@ -101,7 +98,7 @@ const App: React.FC = () => {
     <DndContext sensors={[sensor]} onDragEnd={onDragEnd}>
       <SortableContext items={fileList.map((i) => i.uid)} strategy={verticalListSortingStrategy}>
         <Upload
-          action="https://www.mocky.io/v2/5cc8019d300000980a055e76"
+          action='https://www.mocky.io/v2/5cc8019d300000980a055e76'
```
Comment (afc163) [inline components/upload/demo/drag-sorting.tsx:101]: 本地重装一下 node_modules，这里不应该被 rome 修改。

File: components/upload/demo/drag-sorting.tsx [hunk 2]
```diff
@@ -8,7 +8,6 @@ import {
   verticalListSortingStrategy,
 } from '@dnd-kit/sortable';
 import { CSS } from '@dnd-kit/utilities';
-import { css } from '@emotion/css';
```
Comment (afc163) [inline components/upload/demo/drag-sorting.tsx:11]: package.json 里 @emotion/css 这个是不是可以删掉了？

Comment (MadCcc) [reply components/upload/demo/drag-sorting.tsx:11]: https://github.com/ant-design/ant-design/pull/43623
这里删

File: components/upload/demo/drag-sorting.tsx (hunk 1 above)
Comment (MadCcc) [inline components/upload/demo/drag-sorting.tsx:101]:
```suggestion
          action="https://www.mocky.io/v2/5cc8019d300000980a055e76"
```

**Review by MadCcc — APPROVED** [review]

**Pull request merged**